### DISCUSSION TITLE
关于参考文献不对齐的疑问

### DISCUSSION THREAD

**Discussion by CalvinWaWang** [sjtug/SJTUThesis#978] [提问] [answered]
请问，这样的情况是正常的吗？同一作者的参考文献，为啥间距越来越大额。
<img width="380" alt="截屏2023-09-30 下午6 52 16" src="https://github.com/sjtug/SJTUThesis/assets/33740878/264ba2fa-5e78-481b-9144-540eed3ddc8f">

**Comment by LogCreative** [comment, marked as answer]
正常，因为 LaTeX 默认会对每行的内容采用两端对齐策略。

**Reply by CalvinWaWang** [reply]
感谢！！
